zipkin-server/README.md

```diff
@@ -102,6 +102,7 @@ Attribute | Property | Description
 --- | --- | ---
 environment | zipkin.ui.environment | The value here becomes a label in the top-right corner. Not required.
 defaultLookback | zipkin.ui.default-lookback | Default duration in millis to look back when finding traces. Affects the "Start time" element in the UI. Defaults to 3600000 (1 hour in millis).
+searchEnabled | zipkin.ui.search-enabled | If the Find Traces screen is enabled. Defaults to true.
```

Comment:
we also need to say something about why you'd ever want to do this

Reply:
PTAL cc also @bplotnick